Index: open_file.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- open_file.py	(revision d8e03706898b15ac580e24826f380e17c26c4949)
+++ open_file.py	(date 1589814636227)
@@ -20,10 +20,7 @@
         raise AssertionError
     num_frames = int(len(fp) / frame_dim)
     fp = fp.reshape(num_frames, side_length, side_length)
-    # print(fp)
-    # for i in range(num_frames):
-    #     fp_1 = fp[i]
-    #     cv2.imshow('', fp_1)
+
     return fp
 
 
